onnxruntime/test/python/transformers/benchmark_mha.cmd

```diff
@@ -0,0 +1,47 @@
+echo "Benchmark Scaled Dot Product Attention (SDPA) performance on GPU:"
+
+set CUDA_VISIBLE_DEVICES=0
+python benchmark_mha.py --use_gpu
+python benchmark_mha.py --use_gpu --use_cuda_graph
+python benchmark_mha.py --use_gpu --torch
+
+type benchmark_mha_gpu_*.csv > mha_gpu_benchmark_results.csv
+
+echo "Benchmark performance on CPU with number of threads:"
+set MKL_DYNAMIC=FALSE
+set OMP_NUM_THREADS=1
```

Comment:
isn't the openmp deprecated from onnxruntime?

Reply:
That's for PyTorch